db/migrate/20181026182144_add_tasks_table.rb

```diff
@@ -0,0 +1,13 @@
+class AddTasksTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :tasks, :id => :bigserial do |t|
+      t.string :name
+      t.string :status
+      t.string :state
+      t.string :message
+      t.integer :progress
```

Comment:
Will this be used in the initial implementation?

Reply:
Probably not, initially I expect we just mark the task completed when the service is deployed.